distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Deprecated.enso

```diff
@@ -0,0 +1,12 @@
+import project.Data.Text.Text
+
+## A warning that an API is deprecated.
+type Deprecated
+    ## PRIVATE
+    Warning type_name:Text method_name:Text message:Text=""
+
+    ## PRIVATE
+
+       Pretty prints the Deprecated warning.
+    to_display_text : Text
+    to_display_text self = self.message.if_nothing ("Deprecated Warning: " + self.type_name + "." + self.method_name + " is deprecated.")
```

Comment:
Something's not right here.

Since `message:Text` it cannot ever be `Nothing` - since it would fail to type check and throw a `Type_Error` when such instance is constructed.

So then in `to_display_text`, the `if_nothing` will never evaluate the fallback clause.

I guess we either need
```suggestion
    Warning type_name:Text method_name:Text (message : Text | Nothing = Nothing)

    ## PRIVATE

       Pretty prints the Deprecated warning.
    to_display_text : Text
    to_display_text self = self.message.if_nothing ("Deprecated Warning: " + self.type_name + "." + self.method_name + " is deprecated.")
```

OR

```suggestion
    Warning type_name:Text method_name:Text message:Text=""

    ## PRIVATE

       Pretty prints the Deprecated warning.
    to_display_text : Text
    to_display_text self = if self.message.is_empty.not then self.message else
        "Deprecated Warning: " + self.type_name + "." + self.method_name + " is deprecated."
```

Reply:
Yes meant to do the later. Have fixed.